pandas/core/indexes/category.py

```diff
@@ -186,11 +186,7 @@ def __new__(
         name = maybe_extract_name(name, data, cls)
 
         if is_scalar(data):
-            # don't allow scalars
-            # if data is None, then categories must be provided
-            if data is not None or categories is None:
-                raise cls._scalar_data_error(data)
-            data = []
```

Comment:
This is not actually about passing scalar data, but about passing no data (eg `pd.CategoricalIndex(categories=['a', 'b'])`). We generally support constructors with passing no data to create an empty (eg Series also allows this). 

So I would either keep supporting this, or either it should at least be deprecated (this eg breaks dask)

Reply:
Opened https://github.com/pandas-dev/pandas/issues/38944 to track this